src/test/ui/lifetime-errors/ex3-both-anon-regions-earlybound-regions.stderr

```diff
@@ -1,11 +1,11 @@
-error[E0623]: lifetime mismatch
+error[E0621]: explicit lifetime required in the type of `y`
   --> $DIR/ex3-both-anon-regions-earlybound-regions.rs:17:12
    |
 13 | fn baz<'a, 'b, T>(x: &mut Vec<&'a T>, y: &T)
-   |                               -----      -- these two types are declared with different lifetimes...
+   |                                       - consider changing the type of `y` to `&'a T`
```

Comment:
It seems to me that this test is misnamed. It is called "both-anon-regions", but, in fact, one of the regions is not anomymous. Maybe rename to something like `ex2a-push-one-existing-name-early-bound.rs`? (To be honest, the `exN-` naming scheme is not doing it for me, since it's not clear what list of tests these numbers derive from.)